examples/camera-app/linux/src/clusters/camera-avstream-mgmt/camera-av-stream-manager.cpp

```diff
@@ -329,6 +337,20 @@ void CameraAVStreamManager::OnAttributeChanged(AttributeId attributeId)
         mCameraDeviceHAL->GetCameraHALInterface().SetMicrophoneVolume(GetCameraAVStreamMgmtServer()->GetMicrophoneVolumeLevel());
         break;
     }
```

Comment:
Add the ImageControl settings here to sync the CameraHAL and the SDK attributes

Reply:
Added handling for image control.
Also added attribute handling to OnAttributeStateChanged